modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/ContentContainer/Attachments.tsx

```diff
@@ -159,6 +159,9 @@ export function Attachments({setValues, values}: IProps) {
 
 								setValues({
 									...values,
+									objectDefinitionExternalReferenceCode: String(
```

Comment:
Why are you coercing here? Is there any chance of `emptyStateItem.externalReferenceCode` no being a string primitive? [You just set it to an empty string](https://github.com/liferay-objects/liferay-portal/blob/73d863ff05e58b6972666a2b9740a666ca05d640/modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/AutoComplete/index.tsx#L87). Is there any other part of the code that will change this value? Because, if there is not, calling String() looks unnecessary. Let me know if you think different, I may be overlooking something.